homeassistant/components/google_assistant/http.py

```diff
@@ -125,6 +125,8 @@ def handle_execute(self,
                 domain = eid.split('.')[0]
                 (service, service_data) = determine_service(
                     eid, execution.get('command'), execution.get('params'))
+                if domain == "group":
+                    domain = "light"
```

Comment:
Say you have a group named `group.downstairs_lights`. Won't this send a `set_brightness` service call to `light.downstairs_lights` (which potentially doesn't exist)?

Reply:
I don't know much about the code but the code as I interpreted it sets the domain for the service call but does not include the entity ID domains. I think those are part of the service_data variable.

Reply:
@philk I changed it to use the homeassistant service instead of the light service. Is this okay to merge now?